.github/workflows/check-usings.yml

```
@@ -0,0 +1,71 @@
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v3.2.0
+      with:
+        dotnet-version: 8.0.x
+
+    - name: Install Dependencies
+      run: dotnet restore
+
+    - name: Get Changed Files
+      id: changed-files
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2002:style:3:40: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)